docs/UserGuide.md

```diff
@@ -97,46 +97,50 @@ Expressions without `...` at the end takes the last parameter as the argument wh
 ### Adding a tag with filepath : `tag`
 
 Tags a file with a unique tag name.<br>
-You can add a nickname to a file. Optionally, you can add some labels to the tag for categorisation.
-The `FILE_PATH` field can accept both absolute and relative file path from your current directory in HelloFile.
+You can add a nickname to a file. This make it easier to manage only some specific files that you want such as frequently used files, etc.
```

Comment:
'only' can be omitted

Reply:
I think I'll change the structure of the sentence.